### DISCUSSION TITLE
无代码创建Agent，使用GLM4的企业api_key 和api_base， 问答无回复。 换成个人的api_key就可以了。

### DISCUSSION THREAD

**Discussion by zhangyukun230** [modelscope/ms-agent#442] [Q&A]
我用mofelscope社区搭了一个简单的agent，线上一切正常。本地把代码下载下来后正常运行了 。 模型用的glm4。
但是使用企业的key，需要修改api_key 和base_url
<img width="1309" alt="image" src="https://github.com/modelscope/modelscope-agent/assets/37434447/86bd3ece-3d82-463f-93aa-933fa00bdaf9">
修改后， agent无回复
![image](https://github.com/modelscope/modelscope-agent/assets/37434447/3303ae5b-3528-4cbc-80d7-d491c41304ee)
且无任何报错
换成我个人的glm4 api _key 就没有问题了， agent就可以正常回复

**Comment by zzhangpurdue** [comment]
这里我们在排查了，看看是什么问题。
